packages/scene-composer/src/components/three-fiber/ModelGeometryComponents/PlaneGeometryComponent.tsx

```diff
@@ -61,33 +59,12 @@ const PlaneGeometryComponent: React.FC<PlaneGeometryComponentProps> = ({
     } else if (component.textureUri && !!loadTextureOnMesh) {
       setDirtyTexture(true);
     }
-  }, [meshId, component.textureUri, loadTextureOnMesh, clearTextureOnMesh]);
+  }, [component.textureUri, loadTextureOnMesh, clearTextureOnMesh]);
 
-  return component.isGroundPlane ? (
-    createPortal(
-      <group
-        name={getComponentGroupName(node.ref, 'PLANE_GEOMETRY')}
-        parent={scene}
-        position={node.transform.position}
-        rotation={new THREE.Euler(...node.transform.rotation, 'XYZ')}
-        scale={node.transform.scale}
-      >
-        <Plane
-          args={[component.width, component.height]}
-          ref={meshRef}
-          onClick={onClick}
-          userData={{ nodeRef: node.ref }}
-        >
-          <meshStandardMaterial color={component.color ? component.color : 'undefined'} side={THREE.DoubleSide} />
-        </Plane>
-      </group>,
-      scene,
-      { camera },
-    )
-  ) : (
+  return (
     <group name={getComponentGroupName(node.ref, 'PLANE_GEOMETRY')}>
       <Plane args={[component.width, component.height]} ref={meshRef} onClick={onClick}>
-        <meshStandardMaterial color={component.color ? component.color : 'undefined'} />
+        <meshStandardMaterial color={component.color ? component.color : undefined} />
```

Comment:
Wouldn't this have the same result with letting it dynamically handle `undefined`?
```
<meshStandardMaterial color={component.color} />

Reply:
I think your probably right here.